doc/chicago_police_report.Rmd

```diff
@@ -104,16 +104,16 @@ Our third and final model includes all demographic terms(i.e. gender, race and a
 
 $$ \texttt{complaints_per_year}_i = \beta_0 + \beta_1 \times \texttt{salary_scaled}_i + \beta_2 \times \texttt{gender}_i + \beta_3 \times \texttt{race}_i +  \beta_4 \times \texttt{approx_age}_i + \varepsilon_i $$
 
-The coefficients of this model and their p-values are presented below:
+The coefficients of this model and their p-values are presented below. Please note that the gender `Female` and race `Asian` are the reference categories.
 
 ```{r}
 knitr::kable(readRDS('../results/salary_demographics_reg.rds'), caption = "Table 3. Multivariable Regression Model considering effect of all demographic variables.")
 ```
 
-As seen above, with a significance level of $\alpha = 0.05$, the additional demographic variables have statistically significant coefficients, which implies that they have an effect on the number of complaints. Therefore we should take these potentially confounding variables into account to control for their effect on the number of complaints. This would allow us to examine the true relationship between salary and number of complaints while considering these variables.
+As seen above, with a significance level of $\alpha = 0.05$, the additional demographic variables have statistically significant coefficients, which implies that they have an effect on the number of complaints. Therefore we should take these potentially confounding variables into account to control for their effect on the number of complaints. This would allow us to examine the true relationship between salary and number of complaints while holding the effect of the demographic variables constant. Model 3, which includes all demographic variables, suggests that a 1 unit increase in scaled salary is associated with a 0.0033879 increase in complaint received by a police officer while holding the effects of gender, race and approximate age constant. 
 
-## Limitations
-We acknowledge that our data does not fit all the assumptions of linear regression, therefore, we add advise to interpret the results of our analysis with caution. The assumption of independence is likely not met as individual officers may appear in more than one year. Additionally, it appears that the assumption of linearity is also violated based on Figure 4.
+## Limitations and Future Directions
+We acknowledge that our data does not fit all the assumptions of linear regression, therefore, we advise to interpret the results of our analysis with caution. The assumption of independence is likely not met as individual officers may appear in more than one year. Additionally, it appears that the assumption of linearity is also violated based a visual analysis of Figure 3. We suggest researcher interested in our research question, consider models such as polynomial regression and generalized linear regression with a Poisson or negative binomial link as these models may be better suited to the data.
```

Comment:
Should it be 'researchers interested in our research question' instead of singular researcher?

Reply:
Great catch! Commits down below should fix it up.